test/module/irohad/ametsuchi/peer_query_wsv_test.cpp

```diff
@@ -29,11 +29,14 @@ TEST_F(PeerQueryWSVTest, GetPeers) {
   std::vector<std::shared_ptr<shared_model::interface::Peer>> peers;
   std::shared_ptr<shared_model::interface::Peer> peer1 =
       std::make_shared<shared_model::plain::Peer>(
-          "some-address", shared_model::crypto::PublicKey("some-public-key"));
+          "some-address",
+          shared_model::crypto::PublicKey("some-public-key"),
+          boost::none);
```

Comment:
It could have been possible to avoid this change by making a default value to parameter of `shared_model::plain::Peer`, or defining another constructor without TLS certificate.

Reply:
I am strongly against a different constructor for this purpose, because it would contain no different logic. And I would prefer not specifying a default value because:
- this is only useful in tests to create a peer with hardcoded `none` in certificate.
- it will leave a possibility to forget adding a certificate somewhere in test or prod code. we shall better always see that there is a certificate field in peer. I personally found a missing test case this way.

Reply:
OK, sounds reasonable.